src/applications/accredited-representative-portal/pages/POARequestsPage.jsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
-
+import { useLoaderData } from 'react-router-dom';
 import DigitalSubmissionAlert from '../components/DigitalSubmissionAlert/DigitalSubmissionAlert';
-import POARequestsTableFetcher from '../components/POARequestsTableFetcher/POARequestsTableFetcher';
-import usePOARequests from '../hooks/usePOARequests';
+import POARequestsTable from '../components/POARequestsTable/POARequestsTable';
 
 const POARequestsPage = () => {
+  const { poaRequests } = useLoaderData();
```

Comment:
After the upgrade install, we may want to use a context to allow for updating this list from other places